src/main/java/seedu/command/interpreter/AttendanceCommandInterpreter.java

```diff
@@ -65,43 +68,50 @@ public Command decideCommand(String commandDescription) throws PacException {
         logger.info("My First Log");
         logger.fine("My Second Log");
 
-        switch (commandType) {
+        switch (commandType.toLowerCase()) {
```

Comment:
I use toLowerCase().trim() for all input now, might want to consider that also.

Reply:
Will include that, thankyou!